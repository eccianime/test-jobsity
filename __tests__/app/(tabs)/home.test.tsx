import Home from "@/app/(tabs)/home";
import { render } from "@testing-library/react-native";

jest.mock(
  "@/components/Header",
  () =>
    ({ children }: any) =>
      children,
);
jest.mock("@/components/ComposedSearch", () => jest.fn(() => null));
jest.mock("@/components/home/ShowsList", () => jest.fn(() => null));

describe("Home", () => {
  it("should render Header with Search and ShowsList", () => {
    const { toJSON } = render(<Home />);
    expect(toJSON()).toMatchSnapshot();
  });
});
