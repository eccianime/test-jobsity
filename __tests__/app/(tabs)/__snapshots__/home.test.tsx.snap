// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Header with Search and ShowsList 1`] = `
<View
  className="flex-1 pb-4"
/>
`;
